addons/binding/org.openhab.binding.mysensors/src/main/test/org/openhab/binding/mysensors/test/VariableTest.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mysensors.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Date;
+
+import org.junit.Test;
+import org.openhab.binding.mysensors.internal.exception.RevertVariableStateException;
+import org.openhab.binding.mysensors.internal.sensors.MySensorsVariable;
+import org.openhab.binding.mysensors.internal.sensors.variable.MySensorsVariableVCustom;
+
+public class VariableTest {
+
+    @Test(expected = RevertVariableStateException.class)
+    public void testRevertException1() throws RevertVariableStateException {
+        MySensorsVariable v = new MySensorsVariableVCustom();
+        v.revertValue();
+    }
+
+    @Test(expected = RevertVariableStateException.class)
+    public void testRevertException2() throws RevertVariableStateException {
+        MySensorsVariable v = new MySensorsVariableVCustom();
+
+        v.setValue("1");
+
+        v.revertValue();
+    }
+
+    @Test
+    public void testRevert() throws RevertVariableStateException {
+        MySensorsVariable v = new MySensorsVariableVCustom();
+
+        v.setValue("1");
+        String s = v.getValue();
```

Comment:
I assume that s = "1", but this complete test will still pass if the implementation of v.getValue is `return "0"`

So its not really strong